pkgs/development/tools/gettext-tools/default.nix

```diff
@@ -0,0 +1,105 @@
+
+  configureFlags = [
+     "--disable-csharp" "--with-xz" "--with-installed-libtextstyle"
+  ] ++ stdenv.lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
+    # On cross building, gettext supposes that the wchar.h from libc
+    # does not fulfill gettext needs, so it tries to work with its
+    # own wchar.h file, which does not cope well with the system's
+    # wchar.h and stddef.h (gcc-4.3 - glibc-2.9)
+    "gl_cv_func_wcwidth_works=yes"
+  ];
+
+  postPatch = ''
+   cp ../libtextstyle/m4/libtextstyle.m4 gnulib-m4/
+   substituteInPlace projects/KDE/trigger --replace "/bin/pwd" pwd
+   substituteInPlace projects/GNOME/trigger --replace "/bin/pwd" pwd
+   substituteInPlace src/project-id --replace "/bin/pwd" pwd
+  '' + lib.optionalString stdenv.hostPlatform.isCygwin ''
+    sed -i -e "s/\(cldr_plurals_LDADD = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" src/Makefile.in
+    sed -i -e "s/\(libgettextsrc_la_LDFLAGS = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" src/Makefile.in
+  '';
+
+  nativeBuildInputs = [
+    xz
+    xz.bin
+    autoreconfHook
+  ];
+
+  buildInputs = [
+    gettext-runtime
+    libtextstyle
+  ] # HACK, see #10874 (and 14664)
+    ++ stdenv.lib.optional (!stdenv.isLinux && !stdenv.hostPlatform.isCygwin) libiconv;
+
+  gettextNeedsLdflags = stdenv.hostPlatform.libc != "glibc" && !stdenv.hostPlatform.isMusl;
+
+  enableParallelBuilding = true;
+  enableParallelChecking = false; # fails sometimes
+
+  meta = with lib; {
+    description = "Well integrated set of translation tools and documentation";
+
+    longDescription = ''
+      Usually, programs are written and documented in English, and use
+      English at execution time for interacting with users.  Using a common
+      language is quite handy for communication between developers,
+      maintainers and users from all countries.  On the other hand, most
+      people are less comfortable with English than with their own native
+      language, and would rather be using their mother tongue for day to
+      day's work, as far as possible.  Many would simply love seeing their
+      computer screen showing a lot less of English, and far more of their
+      own language.
+
+      GNU `gettext' is an important step for the GNU Translation Project, as
+      it is an asset on which we may build many other steps. This package
+      offers to programmers, translators, and even users, a well integrated
+      set of tools and documentation. Specifically, the GNU `gettext'
+      utilities are a set of tools that provides a framework to help other
+      GNU packages produce multi-lingual messages.
+    '';
+
+    homepage = https://www.gnu.org/software/gettext/;
+
+    maintainers = with maintainers; [ zimbatm vrthra ];
+    license = licenses.gpl3Plus;
+    platforms = platforms.all;
+  };
+}
+
+// stdenv.lib.optionalAttrs stdenv.isDarwin {
+  makeFlags = [ "CFLAGS=-D_FORTIFY_SOURCE=0" ];
```

Comment:
Please move this inside the main attrset otherwise `overrideAttrs` will not know about this.

```nix
  makeFlags = stdenv.lib.optional stdenv.isDarwin "CFLAGS=-D_FORTIFY_SOURCE=0";
```

Reply:
Make we can try to let ofborg build this with the line commented to see if it still necessary  (introduced [7 years ago](https://github.com/lsix/nixpkgs/commit/961c2fe7c3cb5410b747c258bc29328c0433a264)).